test/queries_generator/queries_generator.cpp

```diff
@@ -0,0 +1,336 @@
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <bits/stdc++.h>
+#define TOTAL_COLUMN 9
+#define NUM_COLUMN_INT 3
+
+using namespace std;
+
+enum Return_type { INTEGER = 0,
+            STRING = 1,
+            TIMESTAMP = 2,
+	    MIX_COL_NUM = 3,
+	    COLUMN = 4,
+	    NUMBER = 5};
+
+auto random_arth_op = [](){std::string op="+-*/";return op[rand()%op.size()];};
+
+auto random_compare_op = []()
+{vector<string> op={">", "<", ">=", "<=", "==", "!="};
+  return op[ rand() % op.size() ];
+};
+
+auto random_date_part = []()
+{vector<string> op={"year", "month", "day", "hour", "minute", "second"};
+  return op[ rand() % op.size() ];
+};
+
+auto random_date_part_extract = []()
+{vector<string> op={"year", "month", "day", "hour", "minute", "second",
+        "timezone_hour", "timezone_minute"};
+  return op[ rand() % op.size() ];
+};
+
+string random_timestamp_string(string& aws_expr)
+{
+    auto year = [](){return rand()%100 + 1900;};
+    auto month = [](){return 1 + rand()%12;};
+    auto day = [](){return 1 + rand()%28;};
+    auto hours = [](){return rand()%24;};
+    auto minutes = [](){return rand()%60;};
+    auto seconds = [](){return rand()%60;};
+    auto fraction_sec = [](){return rand()%1000000;};
+    stringstream timestamp_str;
+
+    timestamp_str << year() << "-" << std::setw(2) << std::setfill('0') << month() << "-" << std::setw(2) << std::setfill('0') << day() << "T" <<std::setw(2) << std::setfill('0') << hours() << ":" << std::setw(2) << std::setfill('0') << minutes() << ":" << std::setw(2) << std::setfill('0') <<seconds() << "." << fraction_sec() << "Z";
+    aws_expr = timestamp_str.str();
+    return aws_expr;
+}
+
+string random_tm_format_string()
+{
+    auto random_format = []()
+    {vector<string> op={"yyyyy ", "yyyy ", "yyy ", "yy ", "y ", "MMMMM ", "MMMM ", "MMM ", "MM ", "M ", "dd ", "d ", "a ", "hh ", "h ", "HH ", "H ", "mm ", "m ", "ss ", "s ", "SSSSSSSSS ", "SSSSSS ", "SSSSS ", "SSS ", "SS ", "S ", "n ", "X ", "XX ", "XXX ", "XXXX ", "XXXXX ", "x ", "xx ", "xxx ", "xxxx ", "xxxxx ", ": ", "- "};
+     return op[ rand() % op.size() ];
+    };
+    int loop = rand() % 10;
+    string frmt;
+    while(loop)
+    {
+        frmt += random_format();
+	loop--;
+    }
+    return frmt;
+}
+
+string random_col(string& aws_expr)
+{
+    int num = 1 + (rand() % NUM_COLUMN_INT);
+    aws_expr = "cast(_" + to_string(num) + " as int)";
+    return "int(_" + to_string(num) + ")";
```

Comment:
in our current status, there is no difference between aws-expr to ceph-expr.
which is the best thing for comparison tests.
same input queries should produce the same output results. 

the "int/float ( ... ) " should remove from the code generator